utils/create-pdf-certificate.ts

```diff
@@ -1,25 +1,26 @@
 /* eslint-disable no-console */
 import fs from 'fs';
-import { createWriteStream } from 'fs';
+import { writeFile } from 'fs/promises';
 import path from 'path';
+import { TDocumentDefinitions } from 'pdfmake/interfaces';
 import { hideBin } from 'yargs/helpers';
 import yargs from 'yargs/yargs';
 
-import { generatePdf, TDocumentDefinitions } from '../packages/generate-pdf/src';
+import { generatePdf } from '../packages/generate-pdf/src';
 import { fileExists, normalizePath } from './helpers';
 
 const fonts = {
-  Lato: {
-    normal: './node_modules/lato-font/fonts/lato-normal/lato-normal.woff',
-    bold: './node_modules/lato-font/fonts/lato-bold/lato-bold.woff',
-    italics: './node_modules/lato-font/fonts/lato-light-italic/lato-light-italic.woff',
-    light: './node_modules/lato-font/fonts/lato-light/lato-light.woff',
+  NotoSans: {
+    normal: `${__dirname}/../../fixtures/fonts/NotoSans-Regular.ttf`,
```

Comment:
I tried running this locally and the fonts were not found, I think the correct path is `${__dirname}/../fixtures/fonts/NotoSans-Regular.ttf`

Reply:
You are right, I updated the path.